src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -283,10 +280,14 @@ public sealed partial class ChaCha20Poly1305 : System.IDisposable
     public enum CipherMode
     {
         CBC = 1,
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         ECB = 2,
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         OFB = 3,
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         CFB = 4,
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         CTS = 5,
```

Comment:
Personally, I'd rather not paint the enum members here, and leave it as a gap in the annotations.  If someone... really... wanted to, they could write an AES implementation that worked for some of these modes.

The joys of polymorphism.